From fe378f2b696cc715d0adc12c4a9f7a0522c50c3a Mon Sep 17 00:00:00 2001
From: sjayanti <satya.charitardha.jayanti@intel.com>
Date: Fri, 20 Jul 2012 14:39:41 +0530
Subject: Last word getting truncated in GPS direction application

BZ: 22016

Original BZ: 10392
MediaPlayerService uses buffer count of 4, this workaround is to give support for
opencore engine added by google. Since we use Stagefright, this is not required.
For betterment of latency changing the buffer count to 2.

Category: aosp improvement
Domain: AUDIO_MEDIA-Audio Framework
Origin: Internal
Upstream-Candidate: no, need rework

Orig-Change-Id: Ib7799715cf2076acfaab73d618adacd8f45010c3

Change-Id: I3ebe94d742a294804fde3da69de5269cdcc07d0a
Signed-off-by: sjayanti <satya.charitardha.jayanti@intel.com>
Reviewed-on: http://android.intel.com:8080/77895
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: M, Arulselvan <arulselvan.m@intel.com>
Tested-by: M, Arulselvan <arulselvan.m@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
Signed-off-by: Arulselvan M <arulselvan.m@intel.com>
---
 media/libmediaplayerservice/MediaPlayerService.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/media/libmediaplayerservice/MediaPlayerService.cpp b/media/libmediaplayerservice/MediaPlayerService.cpp
index cd052e6..d368a9f 100644
--- a/media/libmediaplayerservice/MediaPlayerService.cpp
+++ b/media/libmediaplayerservice/MediaPlayerService.cpp
@@ -198,7 +198,7 @@ static bool checkPermission(const char* permissionString) {
 }
 
 // TODO: Find real cause of Audio/Video delay in PV framework and remove this workaround
-/* static */ int MediaPlayerService::AudioOutput::mMinBufferCount = 4;
+/* static */ int MediaPlayerService::AudioOutput::mMinBufferCount = 2;
 /* static */ bool MediaPlayerService::AudioOutput::mIsOnEmulator = false;
 
 void MediaPlayerService::instantiate() {
-- 
1.7.4.1

